Gems/Atom/RHI/Code/Include/Atom/RHI/DrawItem.h

```diff
@@ -134,7 +134,7 @@ namespace AZ::RHI
             {
                 bool m_enabled : 1;    // Whether the Draw Item should render
```

Comment:
Suggest adding a comment about the union being created to support any number of user flags (as long as they fit within 8bit), but current use only uses a 1-bit bitfield to determine if the draw item is enabled or not.